app/assets/javascripts/uploads.js

```diff
@@ -5,7 +5,7 @@
 //= require blueimp-file-upload/js/jquery.fileupload-ui
 
 // iframe-transport is ajax file upload support for IE, Uncomment if needed      
-// javascript_include_tag  (file_upload + "js/jquery.iframe-transport")      
+// javascript_pack_tag  (file_upload + "js/jquery.iframe-transport")      
```

Comment:
This is not working.
I get `ActionView::Template::Error: undefined method `javascript_pack_tag'`

Reply:
I know, the webpacker integration is not working.

Reply:
There's a problem with the installation itself. The rake task doesn't work after adding `webpacker` to the gemfile.